parm/fv3lam_rrfs.xml

```diff
@@ -399,6 +399,12 @@
        <scale>4.0</scale>
        </param>
 
+       <param>
+       <shortname>SOILM_ON_DEPTH_BEL_LAND_SFC</shortname>
+       <pname>SOILM</pname>
```

Comment:
@EricJames-NOAA Please change SOILM into SOILMOI. The grib2 name 'SOILM' was deprecated in the grib2 tables.